### ISSUE TITLE
Gradient: Improve removing control points

### ISSUE THREAD

**Issue by hanneslsm** [WordPress/gutenberg#50521]
## What problem does this address?
Currently it's not intuitive how to remove a point on a gradient.
You have to select the point on the gradient (aka open the color picker) and find the "Remove Control Point" link on the bottom of the color picker. It's neither a intuitive place for the link, nor a visually easily to distinguish.

<img width="570" alt="image" src="https://github.com/WordPress/gutenberg/assets/17861443/7a490c42-94df-4632-be9e-8dc757fa9ffc">

## What is your proposed solution?
- Removing a point should be possible by dragging the point out of the gradient area. 
- Optional by selecting the point and pressing backspace.

## References
Powerpoint and Figma

https://github.com/WordPress/gutenberg/assets/17861443/8ca88170-4a57-4b08-861e-bdd965f83de9

**Comment by richtabor** [comment]
> You have to select the point on the gradient (aka open the color picker) and find the "Remove Control Point" link on the bottom of the color picker.

This is quite disjointed. 

Can you select and press delete?

**Comment by hanneslsm** [comment]
> Can you select and press delete?

Nope, only way to delete a point is by clicking "Remove Control Point"
